src/styles/ternary.styl

```diff
@@ -19,10 +19,6 @@ h1, h2, h3, h4, h5, h6, p, ul, li, input, form, body, button {
 
 /* Sections */
 
-main:not(header + main) {
```

Comment:
Just to clarify, this bug wasn't caused by this pull right?

Reply:
No. Though for reasons obscure the bug is worse in the pull than in prod.